### DISCUSSION TITLE
Realease

### DISCUSSION THREAD

**Discussion by DanielVilaFalcon** [kubernetes/dashboard#7534] [General]
@maciaszczykm  Could you explain us how to build a container including api and web module in order to get the new fixes?

**Comment by maciaszczykm** [comment]
@DanielVilaFalcon We're working on it with @floreks. It's on top on our priority list and we should be able to move forward with it this Friday. I cannot guarantee any specific date though.

**Comment by floreks** [comment]
The new configuration is still not ready. Setup is a bit more complex than with the old architecture and we'll still need to update the helm and our deployment files. I wouldn't recommend trying to use it just yet.

**Comment by Cell5** [comment]
Same here, having issues with building docker image to fix bug with cronjobs description in k8s v1.25+
Where we needed to BatchV1beta1 to BatchV1 in jobs.go

**Comment by Cell5** [comment]
> @maciaszczykm Could you explain us how to build a container including api and web module in order to get the new fixes?

Hey, find out one of the possible ways to do that through compiling the release 2.7.0 source code.
Download sources and unarchive.
Change code in the **src** directory as needed.
run from the unarchived directory: 
```
yarn
make build
make docker-build-head
```

**Reply by justas200** [reply]
Hi, I cannot seem to find the docker-build-head build configurations

`make: *** No rule to make target 'docker-build-head'.  Stop.`

**Reply by Cell5** [reply]
Hi, this rule is inside the source code archive which you can find attached to the release 2.7.0.
I know it sounds weird but there is no such rule in the master branch Makefile.
Hope it was helpful

**Comment by gergokee** [comment]
Are there any plans to release a new version ? (I cannot find any indication or release plan anywhere.) Thank you very much in advance.

**Comment by maciaszczykm** [comment]
@gergokee We don't have any date yet, sorry. https://github.com/kubernetes/dashboard/pull/7544 needs to be completed first.

**Reply by gergokee** [reply]
Okay, thank you very much for the update.

**Comment by maciaszczykm** [comment]
It's done already.
